app/controllers/import_controller.rb

```diff
@@ -86,26 +86,13 @@ def import
         end
 
         begin
-          # Run validations on fields that have restrictions conditional on other fields
-          validate_required_primary_contact(patient, row_ind)
```

Comment:
although this is commented out for now, is it not validation that we might want to eventually have in the future?

Reply:
This wasn't commented out. But Primary Contact validation is now down on the model instead [here](https://github.com/SaraAlert/SaraAlert/pull/582/files#diff-94dcc014f62f0d9b6084ad98f82f74cd40c86b029ef1691914bdbc960c2f42dcR98)

Reply:
sorry, read the line wrong, thanks for the clarification!